plugins/GSdx/GSHwHack.cpp

```diff
@@ -85,22 +85,22 @@ bool GSC_DBZBT3(const GSFrameInfo& fi, int& skip)
 		}
 		else if(fi.TME && (fi.FBP == 0x00000 || fi.FBP == 0x00e00 || fi.FBP == 0x01000) && fi.FPSM == PSM_PSMCT32 && fi.TPSM == PSM_PSMT8H)
 		{
-			if (Dx_only) { // Ought to be fine with blending accuracy (fbmask?)
+			if (Aggressive) { // Partially works on D3D. OpenGL has no such issues.
 				if(fi.FBMSK == 0x00000)
 				{
-					skip = 28; // outline
-				}
-				if(fi.FBMSK == 0x00FFFFFF)
-				{
-					skip = 1;
+					// Hack removes character outlines as well as black borders on the sides of the screen.
+					// In the past the hack was used to remove red vertical lines on the screen I believe but now that issue has been fixed.
```

Comment:
This line can be removed, as I'm pretty sure the `skip = 1` you removed did that, it shouldn't apply to the remaining hack.

If you want to confirm it you should build the GSdx from v1.4.0 from source after removing the outline hack `skip = 28` and test the CRC hack.